package.json

```diff
@@ -4,14 +4,20 @@
   "version": "0.0.4",
   "author": "Alex Riviere (https://alex.party)",
   "license": "MIT",
-  "main":"dist/geo-elements.iife.js",
+  "main": "dist/geo-elements.iife.js",
   "scripts": {
     "dev": "vite",
     "build": "vite build",
     "build:docs": "vite build --config vite.docs.js ",
-    "serve": "vite --config vite.docs.js"
+    "serve": "vite --config vite.docs.js",
+    "lint": "eslint --config=.eslintrc.js .eslintrc.js vite.config.js vite.docs.js src",
+    "fix": "eslint --fix --config=.eslintrc.js .eslintrc.js vite.config.js vite.docs.js src"
```

Comment:
`npm run lint` to see the errors, `npm run fix` to auto-fix them (where possible)